src/transforms/coltable.jl

```diff
@@ -0,0 +1,14 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    ColTable()
+
+The transform that applies the function `Tables.columntable` to to the input table.
+"""
+struct ColTable <: Stateless end
+
+apply(::ColTable, table) = Tables.columntable(table), table
+
+revert(::ColTable, table, cache) = cache
```

Comment:
```suggestion
revert(::ColTable, newtable, cache) = cache
```

Reply:
@ceferisbarov please pay more attention to the code style. Try to copy/paste the variable names that are used consistently across the transforms. Thank you @eliascarv for spotting that.